non-oss/pharos_pro/commands/license_inspect_command.rb

```diff
@@ -2,5 +2,117 @@
 
 module Pharos
   class LicenseInspectCommand < Pharos::Command
+    options :load_config, :tf_json
+
+    banner "Display information about the installed license on a Pharos Cluster or a license subscription token"
+
+    option '--subscription-token', '[TOKEN]', 'pharos license subscription token'
+    option '--cluster-id', '[ID]', 'pharos cluster id'
+
+    def prepare_config
+      return if @prepared_config
+
+      load_config(master_only: true)
+      cluster_manager('no-generate-name' => true)
+      @prepared_config = true
+    end
+
+    def default_subscription_token
+      prepare_config
+      encoded_jwt = license_secret.data['license.jwt']
+      return nil unless encoded_jwt
+
+      Base64.decode64(encoded_jwt)
+    end
+
+    def default_cluster_id
+      prepare_config
+      cluster_context['cluster-id'] || signal_error('Failed to get cluster id')
+    end
+
+    def execute
+      puts [license_type, owner, valid_until, contact_note].compact
```

Comment:
A lot of stuff had to be added to implement the requested display format in the issue.